include/glow/Backends/DeviceManager.h

```diff
@@ -120,6 +120,12 @@ class DeviceManager {
               std::unique_ptr<ExecutionContext> context,
               runtime::ResultCBTy resultCB) = 0;
 
+  /// Copies the contents of placeholder \p P to the device.
+  virtual Error copyTensorToDevice(Placeholder *P) {
```

Comment:
Shouldn't this also get the Function? 
For backends with placeholders that are bound to networks (instead of being global)

Reply:
Updated this to take in a Placeholder and Tensor, I don't think network name is needed here. If the PH is used by multiple networks we want to update the value for all networks. Also added a cb function to make this an async interface

Reply:
Looks good overall.
Please clarify how should a backend determine if the same PH is being used in multiple networks: name / PH address/ Tensor address?
Maybe this usage would benefit from something more explicit?

Reply:
And one more question, would the RT wait until pending inference have completed before updating such a PH or is that the backend's responsibility?